jenkins/imageDirs.txt

```diff
@@ -1,6 +1,5 @@
 terra-jupyter-base
-terra-jupyter-bioconductor
-terra-jupyter-gatk
-terra-jupyter-hail
 terra-jupyter-python
 terra-jupyter-r
+terra-jupyter-bioconductor
+terra-jupyter-gatk
```

Comment:
do you still want to build gatk now that it's not really working?

Reply:
I'll keep the build since the image builds, but I do not generate documentation. I think its worth keeping since the image seems to work fine, pip is just not in the path correctly (which is what our documentation depends on)